developerportal/apps/events/templates/event.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
```

Comment:
This files' changes are from another PR which was based off this one, which I should have held off merging, but didn't... Sorry.